### DISCUSSION TITLE
Optimal Learning rate

### DISCUSSION THREAD

**Discussion by itsAnanth** [Knightrider16/logistic_regression_from_scratch#4] [Q&A] [answered]
what would be the optimal learning rate for gradient descent based logistic regression for this particular XOR problem

**Comment by Knightrider16** [comment, marked as answer]
0.001. You can experiment with it with your dataset
